### DISCUSSION TITLE
Is there the ability to display the x-axis or y-axis of the victory-chart as a image

### DISCUSSION THREAD

**Discussion by xuanweiH** [FormidableLabs/victory#2113] [Q&A]
Is there the ability to display the x-axis or y-axis of the victory-chart as a image？

**Comment by becca-bailey** [comment]
Can you elaborate on this? As far as I know, you should be able to use some custom component props to display whatever you want inside an axis. For example:

```axisComponent={props => <foreignObject><img src="..." .></foreignObject>```
